### DISCUSSION TITLE
Acknowledgements, abstracts, contentes, chapters and bibliography are starting at even pages

### DISCUSSION THREAD

**Discussion by AndressaO96** [joaomlourenco/novathesis#364] [Q&A] [answered]
Template Version: [v6.10.5]
cloud-based service: no
LaTeX distribution: TEX studio
operating system: Windows 10

Acknowledgements, abstracts, contentes, chapters and bibliography are starting at even pages too. Shouldn't this pages always start on odd pages?
This has been happening at least since February 2022. It's the oldest template version I have and it has this problem.

I notice it today, a day before the dissertation deadline.

![Captura de ecrã 2022-09-29 155025](https://user-images.githubusercontent.com/82115517/193065008-433012e0-6836-4a5c-9cc9-013ae1119cec.png)

**Comment by joaomlourenco** [comment, marked as answer]
@AndressaO96  With the option `media=screen` (default) in "1_novathesis.tex" active, there are no "additional blank pages".
With the option `media=paper` (uncomment the line in "1_novathesis.tex") chapters and important stuff will always start in a odd page (there will be the expected additional blank pages).

**Reply by joaomlourenco** [reply]
You may also add
```latex
\openright
```
in 5_packages.tex.
